### DISCUSSION TITLE
Escape characters with ~ (Orig: Using url in Container 'technology' param)

### DISCUSSION THREAD

**Discussion by SantiagoRevelo** [plantuml-stdlib/C4-PlantUML#328] [Q&A] [answered]
I want to add an API url as the container technology, but I can see an error in the resulting image that adds "</size>//" at the end (see attach)

![Screenshot 2023-11-20 at 12 22 03](https://github.com/plantuml-stdlib/C4-PlantUML/assets/1225185/055e5d2a-170f-4ad4-a85c-d20d4e596fb9)

How can I escape the "//" in order to render the url properly?

**Comment by kirchsth** [comment, marked as answer]
Hi @SantiagoRevelo, 
you can escape it ~
e.g. like 

```plantuml
@startuml
!include <C4/C4_Container.puml>

Container(spa, "SPA", "http:~/~/www.plantuml.com", "Characters can be escaped with ~")
@enduml
```

[![](https://www.plantuml.com/plantuml/svg/FSmn2WCX40NGtbDCjaeWQBDL26635mZa066Y0mhkhEWiTfuzRfFgmt_ylrJ18lkQr2coIxidkDlPsFbZDnQCJ4NdmG-b_iMvPhp2z7uzfn519D-wwQQrfdD2FjwqszO3RS22JgXKSCZm9Q3gC9E75YL0dovABSH-J7u0
)](https://www.plantuml.com/plantuml/uml/FSmn2WCX40NGtbDCjaeWQBDL26635mZa066Y0mhkhEWiTfuzRfFgmt_ylrJ18lkQr2coIxidkDlPsFbZDnQCJ4NdmG-b_iMvPhp2z7uzfn519D-wwQQrfdD2FjwqszO3RS22JgXKSCZm9Q3gC9E75YL0dovABSH-J7u0
)

BR Helmut
